backend/routes.js

```diff
@@ -746,7 +724,7 @@ route.POST('/deleteContract/{hash}', {
   // Authentication passed, now proceed to delete the contract and its associated
   // keys
   try {
-    const [id] = sbp('chelonia.persistentActions/enqueue', ['backend/deleteContract', hash])
+    const [id] = sbp('chelonia.persistentActions/enqueue', ['backend/deleteContract', hash, undefined, true])
```

Comment:
I don't think persistent actions can take non-serializable parameters like `undefined`

Reply:
It'll be coerced into `null`, which I would still work in this case.

Reply:
My preference would be to make this something that's compatible with both the serialization format and the type information, e.g. `false` then. Minor point though, I suppose it can be left as-is

Reply:
It's the way it is due to the argument being optional. I could change this to `null` and update the type information to allow `null`.